example.js

```diff
@@ -0,0 +1,353 @@
+var mutate = require('xtend/mutable')
+var expose = require('choo-expose')
+var css = require('sheetify')
+var html = require('bel')
+var choo = require('./')
+
+css('todomvc-common/base.css')
+css('todomvc-app-css/index.css')
+
+if (module.parent) {
+  exports.todoStore = todoStore
+} else {
+  var app = choo()
+
+  if (process.env.NODE_ENV !== 'production') {
+    var persist = require('choo-persist')
+    var logger = require('choo-log')
+    app.use(persist())
+    app.use(logger())
+  }
+  app.use(expose())
+  app.use(todoStore())
+
+  app.route('/', mainView)
+  app.route('#active', mainView)
+  app.route('#completed', mainView)
+  app.mount('body')
+}
+
+function mainView (state, emit) {
+  emit('log:debug', 'Rendering main view')
+  return html`
+    <body>
+      <section class="todoapp">
+        ${Header(state, emit)}
+        ${TodoList(state, emit)}
+        ${Footer(state, emit)}
+      </section>
+      <footer class="info">
+        <p>Double-click to edit a todo</p>
+        <p>choo by <a href="https://yoshuawuyts.com/">Yoshua Wuyts</a></p>
+        <p>Created by <a href="http://shuheikagawa.com">Shuhei Kagawa</a></p>
```

Comment:
FYI, at first read, I thought this person co-authored choo. My guess is that they wrote the original version of the choo todomvc app.